helios-tools/pom.xml

```diff
@@ -58,6 +58,10 @@
       <groupId>com.typesafe</groupId>
       <artifactId>config</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>jsr311-api</artifactId>
```

Comment:
I think you're right. I'll replace it.

On Fri, Jul 8, 2016 at 9:24 AM, Matt Brown <notifications@github.com> wrote:

> In helios-tools/pom.xml
> <https://github.com/spotify/helios/pull/947#discussion_r70072613>:
>
> > @@ -58,6 +58,10 @@
> >        <groupId>com.typesafe</groupId>
> >        <artifactId>config</artifactId>
> >      </dependency>
> > +    <dependency>
> > +      <groupId>javax.ws.rs</groupId>
> > +      <artifactId>jsr311-api</artifactId>
>
> I could be wrong, but if this dependency is for things like the @GET
> annotation then I think the better dependency is javax.xml.ws:jaxws-api
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/spotify/helios/pull/947/files/b2aee7e1503abf81db1f7041984354ed2169eeea#r70072613>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAdVbco5-cBGJvobMwLW097igY1pRh7wks5qTk-rgaJpZM4JHdpM>
> .
>



-- 
David Xia
Software Engineer
dxia@spotify.com